base/Dockerfile

```diff
@@ -5,8 +5,15 @@ FROM gcr.io/google_appengine/debian8
 RUN apt-get update -y && apt-get install --no-install-recommends -y -q curl python build-essential git ca-certificates libkrb5-dev imagemagick && \
     apt-get clean && rm /var/lib/apt/lists/*_*
 
+# Add the files necessary for verifying and installing node
+# These files are placed in /opt/nodejs so that they will remain
+# even if the /nodejs directory is removed.  Further, the PATH is
+# is not updated, because these scripts should only be called
+# using an absolute path.
+ADD KEYS verify_node verify_install_node /opt/nodejs/
```

Comment:
`/opt/nodejs` doesn't sound like a great name for 'miscellaneous tooling needed for AppEngine runtime image'. Perhaps `/opt/google_appengine/runtime`?

nit: put all these files in a local directory and `ADD` the directory instead of each file individually.
nit: It would be good install_node was added to the same directory too (and a link added from /usr/local/bin). That way all our AppEngine tooling will be in one place.